src/App.js

```diff
@@ -74,29 +74,24 @@ class App extends React.Component {
                         <HomePage {...state} updateData={updateData} />                                  
                     </Route>
                     <Route path='/login'>
-                        <LoginPage {
-                            ...{
-                                buttonText: 'Login',
-                                linkText: 'Logout?',
-                                linkUrl: '/register',
-                                onSubmitLoginForm: this.handlerSubmitLoginForm,
-                                onChangeLoginInput: this.handlerChangeLoginInput
-                            }
-                        } />
+                        <LoginPage                                                        
+                            buttonText={'Login'}
+                            linkText={'Logout?'} 
+                            linkUrl={'/register'}
```

Comment:
it's easier to pass double quoted strings but this way also works, it's up to you